tests/test_datasets.py

```diff
@@ -1,23 +1,50 @@
 """Unit tests for loading Graph dataset."""
 
+import geomstats.backend as gs
+import geomstats.datasets.utils as data_utils
 import geomstats.tests
 from geomstats.datasets.graph_data_preparation import Graph
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.geometry.special_euclidean import SpecialEuclidean
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
 
 
-class TestLoadGraph(geomstats.tests.TestCase):
+class TestDatasets(geomstats.tests.TestCase):
     """Test for loading graph-structured data."""
 
-    @geomstats.tests.np_only
     def setUp(self):
-        """Declare the graph by default and the Karate club graph."""
+        """Set up tests."""
         self.g1 = Graph()
         self.g2 = Graph(
-            graph_matrix_path='data'
-                              '/graph_karate/karate.txt',
-            labels_path='data'
-                        '/graph_karate/karate_labels.txt')
+            graph_matrix_path=data_utils.KARATE_PATH,
+            labels_path=data_utils.KARATE_LABELS_PATH)
 
-    @geomstats.tests.np_only
+    def test_load_cities(self):
+        """Test that the cities coordinates belong to the sphere."""
+        sphere = Hypersphere(dim=2)
+        data, _ = data_utils.load_cities()
+        data = sphere.spherical_to_extrinsic(data)
+        result = sphere.belongs(data)
```

Comment:
Another comment: `spherical_to_extrinsic` always returns a norm-1 vector by construction, so the belong check doesn't really check anything. It doesn't really matter here because you just want to make sure load_cities ran smoothly.